exporter/opentelemetry-exporter-jaeger-thrift/src/opentelemetry/exporter/jaeger/thrift/send.py

```diff
@@ -96,19 +96,29 @@ def emit(self, batch: jaeger.Batch):
 
 
 class Collector:
-    """Submits collected spans to Thrift HTTP server.
+    """Submits collected spans to Jaeger collector in jaeger.thrift
+    format over binary thrift protocol. This is recommend option in cases where
+    it is not feasible to deploy Jaeger Agent next to the application,
+    for example, when the application code is running as AWS Lambda function.
+    In these scenarios the Jaeger Clients can be configured to submit spans directly
+    to the Collectors over HTTP/HTTPS.
 
     Args:
-        thrift_url: URL of the Jaeger HTTP Thrift.
+        collector_endpoint: Endpoint used to send spans
+            directly to Collector the over HTTP.
         auth: Auth tuple that contains username and password for Basic Auth.
+        timeout_in_millis: timeout for THttpClient.
     """
 
-    def __init__(self, thrift_url="", auth=None):
-        self.thrift_url = thrift_url
+    def __init__(self, collector_endpoint, auth=None, timeout_in_millis=None):
```

Comment:
Was this change intentional? Based on the change description, i wouldnt expect the interface to the Collector class to change.

Reply:
I was under the impression that only `JaegerExporter` shouldn't make changes like this although technically the `Collector` and `AgentClientUDP` are not private . I wonder if we should make them private. It's not like `Collector` is configurable in exporter so I wouldn't expect people use this directly. Thoughts?

Reply:
If can make `Collector` private, it would be a breaking change.
If we change `def __init__(self, thrift_url="", auth=None):` to `def __init__(self, collector_endpoint, auth=None, timeout_in_millis=None):` it would be a breaking change as well.

What is making this change breaking is changing the first argument from `thrift_url=""` to `collector_endpoint`. It can be easily avoided just by not making this change but I understand, having a `thrift`-named argument is undesirable, but that is precisely the price to pay for backwards compatibility, right? :shrug:

Reply:
I agree, but `thirft_url=""` especially the empty string was annoying so I thought of changing it because Collector is not something end user uses directly. I will revert this back.